<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gandom_dialog</class>
 <widget class="QDialog" name="Gandom_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lvl_fgandom_l">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="nzirkesht_l">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lvl</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upgrade_zaming_butt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius:40px;
color: rgb(120, 60, 0);
background-color: rgb(255, 201, 66);</string>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
  </widget>
  <widget class="QLabel" name="capacity_gandom_l">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Under crop cultivation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>capacity</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="tedad_barayekashtspinBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 201, 66);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kashtg_butt">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius:40px;
color: rgb(120, 60, 0);
background-color: rgb(255, 201, 66);</string>
   </property>
   <property name="text">
    <string>Gardening</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bardasht_butt">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jokerman</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-radius:40px;
color: rgb(120, 60, 0);
background-color: rgb(255, 201, 66);</string>
   </property>
   <property name="text">
    <string>scythe</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
